ecommerce/src/main/kotlin/io/hhplus/ecommerce/config/RedisConfig.kt

```
@@ -19,6 +23,37 @@ import org.springframework.data.redis.serializer.StringRedisSerializer
 @Configuration
 class RedisConfig {
 
+    @Value("\${spring.data.redis.host:localhost}")
+    private lateinit var redisHost: String
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)